pkgs/applications/editors/vim/plugins/vim-plugin-names

```diff
@@ -937,6 +937,7 @@ https://github.com/xiyaowong/transparent.nvim/,HEAD,
 https://github.com/Wansmer/treesj/,main,
 https://github.com/tremor-rs/tremor-vim/,,
 https://github.com/cappyzawa/trim.nvim/,,
+https://github.com/simonclean/triptych.nvim/,HEAD,
```

Comment:
The URL added to `vim-plugin-names` contains a typo:

```suggestion
https://github.com/simonmclean/triptych.nvim/,HEAD,
```

Also, this PR does not update `generated.nix`, so it doesn't actually add the package to `nixpkgs`.